packages/modular-scripts/src/__tests__/app.test.ts

```diff
@@ -197,7 +197,7 @@ describe('When working with a nested app', () => {
     ).toMatchSnapshot();
   });
 
-  it('can generate a css/main.a0f92c83.chunk.css', async () => {
+  it('can generate a css/main.1a7488ce.css', async () => {
```

Comment:
This and subsequent tests have the file name hard coded in the tests - can we make this dynamic?

Reply:
Agree that we should make them decoupled from the hashes (for example: "it can generate a style hashed main chunk in the css directory"). Not sure if you mean that with dynamic?

Reply:
New issue created: #1481